### DISCUSSION TITLE
Is there any plans to support Into<Outcome> for Result and Optional?

### DISCUSSION THREAD

**Discussion by hasunwoo** [rwf2/Rocket#2150] [Ideas] [answered]
I noticed that i couldn't use try(?) operator on Outcome type when i try to implement FromRequest for my own type. Is there any plans to provide impl Into<Outcome> for Result and Optional?

**Comment by SergioBenitez** [comment, marked as answer]
It's not quite so simple. What you're asking for is not possible on Rust stable. To circumvent, we have the `try_outcome!` macro which does the same thing ? would do. See #597 for more.
